bridge/engine/ActiveMediaList.cpp

```diff
@@ -89,11 +91,10 @@ bool ActiveMediaList::addAudioParticipant(const size_t endpointIdHash)
         _dominantSpeaker = endpointIdHash;
     }
 
-    if (_audioSsrcRewriteMap.size() == _maxActiveListSize)
+    if (_audioSsrcRewriteMap.size() == _audioLastN)
```

Comment:
Why do we not just add audio sources until we run out of audio ssrcs. Why stop at "top3"